tests/kernel/userbufer_validate/src/main.c

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ztest.h>
+extern void test_buffer_rw_read(void);
+extern void test_buffer_writeable_write(void);
+extern void test_buffer_readable_read(void);
+extern void test_buffer_readable_write(void);
+extern void test_buffer_supervisor_rw(void);
+extern void test_buffer_supervisor_w(void);
+extern void test_buffer_user_rw_user(void);
+extern void test_buffer_user_rw_supervisor(void);
+extern void test_multi_page_buffer_writeable(void);
+extern void test_multi_page_buffer_readable(void);
+extern void test_multi_page_writeable(void);
+extern void test_multi_page_user_supervisor_rw(void);
+extern void test_multi_page_supervisor_rw(void);
+extern void test_multi_page_user_rw(void);
+extern void test_multi_page_supervisor_rw_user(void);
+extern void test_multi_page_supervisor_rw_supervisor(void);
+/*test case nain entry*/
+void test_main(void *p1, void *p2, void *p3)
+{
+		ztest_test_suite(userbuffer_validate,
```

Comment:
1 tab only for indentation on all these lines.

Reply:
1 tab indentation has been corrected.